model/gen-ai/spans.yaml

```diff
@@ -71,6 +71,18 @@ groups:
         requirement_level: recommended
       - ref: gen_ai.usage.output_tokens
         requirement_level: recommended
+      - ref: gen_ai.system.instructions
```

Comment:
we should have a ref to this in the future and maybe right now

Reply:
there is a template and template substitution, can we define attributes for those and then record less and more structured